File: sdk/src/main/java/com/cxense/cxensesdk/ApiError.java
package com.cxense.cxensesdk;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

/**
 * Api Error answer
 *
 * @author Dmitriy Konopelkin (dmitry.konopelkin@cxense.com) on (2017-06-05).
 */

final class ApiError {
    @Nullable
    @SerializedName("error")
    String error;
}

File: sdk/src/main/java/com/cxense/cxensesdk/model/ApiError.kt
package com.cxense.cxensesdk.model

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

/**
 * Api Error answer
 *
 */
@JsonClass(generateAdapter = true)
class ApiError(
    @Json(name = "error") val error: String? = null
)
